Casks/pe-client-tools-2019.8.rb

```diff
@@ -0,0 +1,25 @@
+cask 'pe-client-tools-2019.8' do
+  case MacOS.version
+  when '10.13'
```

Comment:
Is 10.12 also supported?

Reply:
I listed the three OSs that are on https://puppet.com/try-puppet/puppet-enterprise-client-tools/ but am happy to adjust as you all see fit :)